recipes/r-cerebroprepare/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+    name: r-cerebroprepare
+    version: 1.0.0
+
+source:
+    git_url: https://github.com/romanhaa/cerebroPrepare.git
+    sha256: 21eb9658ec53c33792feafad3970b4be1d4e3a77d537a6b7e4bca51ad504012a
+
+build:
+    number: 0
+    noarch: generic
+    script: $R CMD INSTALL --build .
+
+rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+    run:
+        - r-base
+        - r-rcpp
+        - r-biocmanager >=1.30.4
+        - bioconductor-biomart >=2.38.0
+        - r-dplyr >=0.7.8
+        - r-future.apply >=1.1.0
+        - r-httr >=1.4.0
+        - r-matrix >=1.2
+        - r-pbapply >=1.4
+        - r-readr >=1.3.1
+        - r-seurat >=2.3.0
+        - r-tidyr >=0.8.2
+
+    host:
+        - r-base
+        - r-rcpp
+        - r-biocmanager >=1.30.4
+        - bioconductor-biomart >=2.38.0
+        - r-dplyr >=0.7.8
+        - r-future.apply >=1.1.0
+        - r-httr >=1.4.0
+        - r-matrix >=1.2
+        - r-pbapply >=1.4
+        - r-readr >=1.3.1
+        - r-seurat >=2.3.0
+        - r-tidyr >=0.8.2
+
+test:
+    commands:
+        - $R -e "library(cerebroPrepare); help("getEnrichedPathways")"
+
+about:
+    home: https://github.com/romanhaa/cerebroPrepare
+    doc_url: https://www.biorxiv.org/content/10.1101/631705v1
```

Comment:
Not sure I'd count a paper as docs for a package. You can put DOIs into extra - that's where we'd expect the paper. It will be linked on the package web page.